.pre-commit-config.yaml

```diff
@@ -25,3 +25,8 @@ repos:
         files: \.(yaml|yml)$
         types: [file, yaml]
         entry: yamllint --strict -c yamllint-config.yaml
+
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.7.1
+    hooks:
+      - id: prettier
```

Comment:
This is not using the configuration you've mentioned. Please read the pre-commit `prettier` hook docs and see the CI logs:
![image](https://user-images.githubusercontent.com/7453394/200939516-3ce95dbb-ba05-4e9f-a5e9-64c109bf9956.png)

Reply:
Totally missed that, sorry about that working on fixing it.